### PR TITLE
[CMS-799][BUGS-5125] Release notes for pantheon-cache WPCLI command deprecation, REST API responses

### PR DESCRIPTION
**Pull request by jazzsequence**
## Summary

**[Pantheon WordPress Upstream](https://pantheon.io/docs/start-states/wordpress)** - adds release notes for change in pantheon-systems/WordPress#336

### Dependencies and Timing

- [x] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [x] After pantheon-systems/WordPress#336 is merged

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7599](https://pr-7599--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `44` | `56.00000000000001` | `12` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7599` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/40658/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/40658/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7599--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by EdwardAngert** [comment]
thanks @jazzsequence (and @jspellman814 )!
it doesn't looks like we have docs documentation on `wp pantheon-cache` so I guess we're ready for this one whenever you are
the doc I was expecting to find more info in, [WordPress Pantheon Cache Plugin Configuration](https://pantheon.io/docs/wordpress-cache-plugin) hasn't seen a content update in something like 7 years(!)

Should we have someone work through this guide and check it over, or are things mostly the same? It doesn't have to be a part of this PR - just went looking for the command and noticed the "years ago" in the existing doc's blame

File: source/content/start-states/wordpress.md [hunk 1]
```diff
@@ -14,6 +14,16 @@ For the most part, [Pantheon's WordPress upstream](https://github.com/pantheon-s
 
 ## Latest Release
 
+### 2022-08-16
+
+#### <a name="20220816" class="release-update"></a>Deprecate `pantheon-cache` in favor of `pantheon cache`
+
+When the [Pantheon Advanced Page Cache](https://wordpress.org/plugins/pantheon-advanced-page-cache/) plugin is active, WordPress sites on Pantheon have two distinct but similarly-named WP-CLI commands: `wp pantheon cache` (which is provided by Pantheon Advanced Page Cache) and `wp pantheon-cache` (which is provided by the Pantheon mu-plugin. This creates in a confusing end result where it's unclear what Pantheon cache WP-CLI command to use or which subcommands are available for which parent commands.
```
Comment (EdwardAngert) [inline source/content/start-states/wordpress.md:21]:
```suggestion
When the [Pantheon Advanced Page Cache](https://wordpress.org/plugins/pantheon-advanced-page-cache/) plugin is active, WordPress sites on Pantheon have two distinct but similarly-named WP-CLI commands: `wp pantheon cache` (which is provided by Pantheon Advanced Page Cache) and `wp pantheon-cache` (which is provided by the Pantheon mu-plugin. This leads to confusion about which Pantheon cache WP-CLI command to use or which subcommands are available for which parent commands.
```

**Comment by jazzsequence** [comment]
It doesn't look like anything in that doc has changed, however I wasn't able to find any documentation about the WP-CLI commands in either the `mu-plugin` (which this update modifies) or in the Pantheon Advanced Page Cache plugin. I think it's probably worth creating a doc for _all_ the WP-CLI commands available (with a note that some are only available if the PAPC plugin is installed and active).

**Comment by jazzsequence** [comment]
@EdwardAngert pantheon-systems/WordPress#336 has been merged, so this is good to go now.

File: source/content/start-states/wordpress.md [hunk 2]
```diff
@@ -14,6 +14,16 @@ For the most part, [Pantheon's WordPress upstream](https://github.com/pantheon-s
 
 ## Latest Release
 
+### 2022-08-16
```
Comment (jspellman814) [inline source/content/start-states/wordpress.md:17]: I imagine we'll hold this update for the next WP core release, so this date will likely change. That's a product decision for @scottbuscemi 😄

Comment (jazzsequence) [reply source/content/start-states/wordpress.md:17]: The [previous update](https://pantheon.io/docs/start-states/wordpress#2022-07-12) didn't necessarily align with a WordPress release (or, if it did, it wasn't noted), so I was following that precedent.

Comment (jazzsequence) [reply source/content/start-states/wordpress.md:17]: In fact, none of the other updates in the release notes mention a WordPress version besides 6.0 (when themes broke), but I guess the change doesn't actually get applied until a WordPress release goes out.

Comment (EdwardAngert) [reply source/content/start-states/wordpress.md:17]: just confirming that this PR is still good to go and that this convo here is "in the future"

🚀 👍 ?

Comment (jazzsequence) [reply source/content/start-states/wordpress.md:17]: We actually chatted about this on our retro today. We need to have another conversation about it when @scottbuscemi is around, so let's hold off for now (unless there's a WordPress release that sneaks out that the change would go to users with).

**Review by EdwardAngert — APPROVED** [review]

**Pull request merged**